lib/ret/hub.ex

```diff
@@ -300,41 +360,70 @@ defmodule Ret.Hub do
 end
 
 defimpl Canada.Can, for: Ret.Account do
+  alias Ret.{Hub}
+  @object_actions [:spawn_and_move_media, :spawn_camera, :spawn_drawing, :pin_objects]
```

Comment:
I don't really have a strong preference, but maybe I'm not sure what you had in mind exactly for the DRY'ed version. Since you're looking at this code for multiple owners, feel free to change it.